keyless-entry

```diff
@@ -273,6 +283,20 @@ def enable_always(args, once=False):
         sys.exit('Already enabled')
     if not filecmp.cmp(crypttab_file, keyful_file):
         sys.exit(f'{crypttab_file} is different from {keyful_file}, aborting')
+    if not os.path.exists(crypttab_file):
+        sys.exit(f'{crypttab_file} does not exist')
+    # This will abort if /etc/crypttab is missing or has unexpected format.
+    filesystems, keyless_content = make_keyless_content(crypttab_file)
```

Comment:
It appears that the only thing you're calling `make_keyless_content` here for is to get the list of filesystems, but that list of filesystems is already stored in the config by `configure`, so you should get the list from there rather than calling `make_keyless_content` again, which is confusing.

It's true that the config file is currently only storing the source for each filesystem and not the target. I would suggest modifying `configure` to store the target in `target{fs_num}` settings just as it is storing the sources, so those can be fetched from there and used  the "Adding key to {target}" message here (which is the only thing the targets are being used for here).

Reply:
I've changed it to use a for loop with the keys from the source and new target entries in the config file.